dev/tasks/tasks.yml

```
@@ -1098,6 +1098,14 @@ tasks:
         UBUNTU: 20.04
       image: ubuntu-cpp-bundled
 
+  test-ubuntu-24.04-cpp-bundled-offline:
+    ci: github
+    template: docker-tests/github.linux.yml
+    params:
+      env:
+        UBUNTU: 24.04
+      image: ubuntu-cpp-bundled-offline
```

Increase timeout to see how long it takes in order to find a suitable timeout:
```suggestion
      image: ubuntu-cpp-bundled-offline
      timeout: 120
```